src/js/step.js

```diff
@@ -260,7 +260,7 @@ export class Step extends Evented {
     const labelId = `${this.id}-label`;
 
     this.shepherdElementComponent = new ShepherdElement({
-      target: document.body,
+      target: this.tour.options.defaultStepOptions.elementContainer || document.body,
```

Comment:
If this is a tour property, it should go on the tour. It will also need docs and tests.

Reply:
> It will also need docs and tests.

@rwwagner90 can you give some intro guidance as to what you'd like to see here?